test/unit/parse/param.test.js

```diff
@@ -5,37 +5,43 @@ function makeSpec() {
   return new Map()
 }
 
-test('minimal', (t) => {
+test('it should ignore the param if the name is empty', t => {
+  const spec = makeSpec()
+  param({ name: null }, spec)
+  t.deepEqual(spec, new Map())
+})
+
+test('it should use an empty object when only the name is given', t => {
   const spec = makeSpec()
   param({ name: 'search' }, spec)
   t.deepEqual(spec, new Map().set('search', new Set([{}])))
 })
 
-test('value', (t) => {
+test('it should set the value of params when a value is given', t => {
```

Comment:
I updated a bunch of these test names, just to make them more descriptive. Didn't have the patience to update them everywhere I made changes though.